tools/EMLassemblyline/make_eml.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="makeeml" name="Make EML" version="0.1.0+galaxy0">
+    <description> Create EML from EAL templates</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_template &&
+    cd output_template &&
+    #for $input in $templates
+        cp '$input' '${input.element_identifier}.txt' &&
+    #end for
+    find . -type f -empty -print -delete &&
+    Rscript '$__tool_directory__/make_eml.R'
+    ]]></command>
+    <inputs>
+        <param name="templates" type="data_collection" type_collection="list" format="tabular" label="Upload all EAL templates" help= "Input a data collection." />
+    </inputs>
+    <outputs>
+        <data name="metadata" format="xml" label="Metadata">
+            <discover_datasets directory = "output_template" pattern= "(?P&lt;designation&gt;.+)\.xml" visible="true"/>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1"> 
+            <param type="data_collection" name="templates" value="Assessing_the_importance_of_field_margins_for_bat_species.xml">
+                <collection type="list">

```

Comment:
Is it __always__ those 7 files? Or can be more, or less?

Reply:
It is depending the content of the input file